fluidBuild.config.cjs

```diff
@@ -198,6 +198,7 @@ module.exports = {
 			],
 			"no-js-file-extensions": [
 				// PropertyDDS uses .js files which should be renamed eventually.
+				".*/PropertyDDS/.*",
```

Comment:
Merge "conflict", the new version of this line is the next one, with explicit `experimental/`. We can remove this one.